### DISCUSSION TITLE
General Ignite / Infinitered appreciation post

### DISCUSSION THREAD

**Discussion by jvgeee** [infinitered/ignite#1771] [General]
May be off-topic or not the right place for it, but I just wanted to say THANK YOU to everyone involved in this project. I've been using Ignite for a few months as a boilerplate, and it's made it so easy for me and my team to get an MVP up and running without worrying about the general fuckery of setting up navigation, stores, directory trees etc. 

Really love the generators too, making a component or a model is so damn **_easy_**. I love that I can customise the templates for these too. And the theme system for importing colours and typography is a really nice developer experience.  

Anyway, just wanted to say thanks. It's an awesome boilerplate, clearly the best for React Native, and I'm really impressed with the amount of support you guys are giving. Legends!

**Comment by jamonholmgren** [comment]
@jvgeee I somehow missed this (still getting used to Github Discussions TBH) -- sorry about that!

This is incredibly kind of you to post! I'm very happy that Ignite has been useful to you and your team. That's exactly the kind of impact that we are always striving for. It makes the time spent well worth it.

![harrison-bader-birds-on-the-black-1](https://user-images.githubusercontent.com/1479215/133183473-0a74b5f1-eace-4fac-94a8-b8b3ae3ff5cd.gif)
